TFLite/src/main.py

```diff
@@ -36,21 +41,49 @@ def get_duplication(tflite_parser):
             else:
                 duplicated_edges += 1
 
+    print("Duplication stats")
     print("Total nodes:", total_nodes, "Duplicated nodes:", duplicated_nodes)
-    print("Total edges:", total_edges, "Duplicated edges:", duplicated_edges)
-
-if __name__ == "__main__":
+    print("Total traversed edges:", total_edges, "Duplicated traversed edges:", duplicated_edges)
+    print()
 
+# Function to load data in models/filename to spanner DB
+def load_data(filename, model_name, category):
     tflite_parser = TFLiteParser.TFLiteParser()
+    graph = tflite_parser.parse_graph("models/" + filename, model_name, category)
```

Comment:
Let's have either a CLA with default value of constant literal for "models" directory. Becomes easier to change in once place if any change needed.